### DISCUSSION TITLE
Use already existing calendar

### DISCUSSION THREAD

**Discussion by Ben-Bitdiddle-DE** [mampfes/hacs_waste_collection_schedule#2695] [Q&A] [answered]
Hello,
i already have an existing HA calendar entity which contains all the waste schedule entries (in fact, this calendar already has been populated with the original providers ICS file, but outside/separate from HA itself), so this is no static file nor a provider URL to be configured. Is there a way to make use of this existing calendar? (By configuring the provider URL i get an additional waste calendar which is kind of redundant then).
Thanks,
KR

![image](https://github.com/user-attachments/assets/985c0a3c-23c0-440b-a59f-80f9e1e72e1c)

**Comment by 5ila5** [comment, marked as answer]
No, you cannot use an existing calendar entity as input for WCS, WCS always creates its own calendar entity.
